package.json

```diff
@@ -43,5 +43,6 @@
   "workspaces": [
     "applications/*",
     "theia-extensions/*"
-  ]
+  ],
+  "dependencies": {}
```

Comment:
I understand that changes in this file were temporarily added by executing `npx browserslist@latest --update-db`, but we can now revert these updates now that the `yarn.lock` was upgraded. You can `amend` the pull-request and revert changes made to `package.json`.

Reply:
ok